--- OpenSURF_version1c/SubFunctions/FastHessian_interpolateExtremum.m	2010-11-28 16:48:53.000000000 +1000
+++ FastHessian_interpolateExtremum.m	2010-11-28 19:48:11.000000000 +1000
@@ -34,6 +34,8 @@
     ipts(np).y = double(((r + O(2))) * t.step);
     ipts(np).scale = double(((2/15) * (m.filter + O(3) * filterStep)));
     ipts(np).laplacian = fix(FastHessian_getLaplacian(m,r,c,t));
+    % added by PIC
+    ipts(np).strength = FastHessian_getResponse(m,r, c, t);
 end
   
 function D=FastHessian_BuildDerivative(r,c,t,m,b)
